libs/interpret/src/lib/MLIDashboard/Controls/GlobalExplanationTab/GlobalExplanationTab.tsx

```diff
@@ -438,7 +438,7 @@ export class GlobalExplanationTab extends React.PureComponent<
     // set to dependence plot initially, can be changed if other feature importances available
     const xKey = JointDataset.DataLabelRoot + featureIndex.toString();
     const xIsDithered =
-      this.context.jointDataset.metaDict[xKey].treatAsCategorical;
+      this.context.jointDataset.metaDict[xKey]?.treatAsCategorical;
```

Comment:
Is there a unit test for this change? In what situations can treatAsCategorical be not available as a property?

Reply:
In this case when xKey isn't available, treatAsCategorical will be undefined. @imatiach-msft [imatiach-msft] or @riedgar-ms  any idea when is xKey not available?

Reply:
hmm, no idea.  It's unfortunate we don't have a repro of this issue...

Reply:
Is there any logging we can usefully do for when the 'null' case is hit? Or are the usual values of xKey considered customer data?

Reply:
are there other places in the code that we need to fix knowing that `treatAsCategorical` can be optional?

Reply:
Thanks for fixing this everywhere.

Reply:
Thanks for catching and commenting about this!! :)